rasa/cli/evaluate.py

```diff
@@ -139,6 +147,10 @@ def _run_markers(
             "Please see errors listed above and fix before running again."
         )
 
+    # Subtract one to remove the virtual OR over all markers
+    num_markers = len(markers) - 1
```

Comment:
this is the total number of all conditions and operators used in all marker configurations that were evaluated -- to get the number of user-defined markers we should
1. get rid of the special case [here](https://github.com/RasaHQ/rasa/blob/main/rasa/core/evaluation/marker_base.py#L450) and always add an "ANY_MARKER"
2. `len(markers.sub_markers)` instead of `len(markers)`

Reply:
@usc-m , I could help and add that while you work on other comments if you like?

Reply:
Yeah, I suggested that too, also the branching factor (maximum number of children under any marker)